sdc/tests/tests_perf/test_perf_read_csv.py

```diff
@@ -43,12 +43,8 @@ def generate_csv(rows=10**5, columns=10):
     file_name = f"data_{rows}.csv"
     r = random.Random(0)  # seed=0
     generate(rows,
-        ['A', 'B', 'C'],
-        [
-            lambda: int(r.random() * 10000),
-            lambda: r.uniform(-1.0, 1.0),
-            lambda: r.uniform(-1.0, 1.0)
-        ],
+        [f"f{c}" for c in range(columns)],
+        [lambda: r.uniform(-1.0, 1.0) for _ in range(columns)],
```

Comment:
Why not just call `r.uniform(-1.0, 1.0)` here without lambda?
`[r.uniform(-1.0, 1.0) for _ in columns]`

Reply:
Because it is generators, not values.